frontend/src/hooks/useLookupCodes.tsx

```diff
@@ -63,7 +63,7 @@ export function useCodeLookups() {
     const classifications = getByType(API.PROPERTY_CLASSIFICATION_CODE_SET_NAME);
     return filter
       ? (classifications ?? []).map(c => mapLookupCode(c)).filter(filter)
-      : !keycloak.hasClaim(Claims.ADMIN_PROPERTIES)
+      : !keycloak.hasClaim(Claims.PROPERTY_EDIT)
```

Comment:
This probably isn't ideal as all users essentially will be able to select Demolish, Subdivide, and Disposed.  Are other users allowed to do this?  I'm pretty sure they definitely can't Dispose outside of the Disposal Project workflow.